pkg/synchromanager/clustersynchro/queue/event_test.go

```diff
@@ -0,0 +1,137 @@
+package queue
+
+import (
+	"testing"
+)
+
+type pressureEventsTest struct {
+	name       string
+	older      *Event
+	newer      *Event
+	reputCount int
+}
+
+func TestPressureEvents(t *testing.T) {
+	tests := []pressureEventsTest{
+		{
+			name: "Newer event is not nil",
+			older: &Event{
+				reputCount: 1,
+				Action:     Added,
+				Object:     "Object1",
+			},
+			newer: &Event{
+				reputCount: 2,
+				Action:     Updated,
+				Object:     "Object2",
+			},
+			reputCount: 2,
```

Comment:
We might be better off with the full event as expected, even though the data and reputCount are the same as the newer, but the Action might change, for example here the event Action would be `Added`

Reply:
I am not sure if I understood it correctly.Do I have to add more tests as well?

Reply:
I think we can use an expected event object to compare, e.g.
```go
			name: "Newer event is not nil",
			older: &Event{
				reputCount: 1,
				Action:     Added,
				Object:     "Object1",
			},
			newer: &Event{
				reputCount: 2,
				Action:     Updated,
				Object:     "Object2",
			},
			expected: Event{
				reputCount: 2,
				Action:     Added,
				Object:     "Object2",
			},
````